CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+# How to Contribute to BotKube
+
+We'd love your help!
+
+BotKube is [MIT Licensed](LICENSE) and accepts contributions via
+GitHub pull requests. This document outlines some of the conventions
+on development workflow, commit message formatting, contact points and
+other resources to make it easier to get your contributions accepted.
+
+We gratefully welcome improvements to
+[documentation](https://www.botkube.io/ "Go to documentation site") as
+well as to code.
+
+## Contributing to documentation
+You can contribute to documentation by following [these
+instructions](https://github.com/infracloudio/botkube-docs#contributing
+"Contributing to BotKube Docs")
+
+## Compile BotKube from source code
+### Prerequisite
+* Make sure you have `go` compiler installed.
+* BotKube uses [`dep`](https://github.com/golang/dep) to manage
+dependencies.
+* You will also need `make` and
+[`docker`](https://docs.docker.com/install/) installed on your
+machine.
+* Clone the source code
+   ```sh
+   $ git clone https://github.com/infracloudio/botkube.git $GOPATH/src/github.com/infracloudio/botkube
+   $ cd $GOPATH/src/github.com/infracloudio/botkube
+   ```
+
+Now you can build and run BotKube by following one of the following
+ways
+### Build the container image
+1. This will build BotKube and create a new container image tagged as `infracloud/botkube:latest`
+   ```sh
+   $ make build
+   $ docker tag infracloud/botkube:latest your_account/botkube:latest
+   $ docker push your_account/botkube:latest
+   ```
+   Where your_account is Docker hub account to which you can push the image
+
+2. Follow the [instructions from
+   README.md](https://github.com/infracloudio/botkube#using-helm) to
+   deploy newly created image in your cluster.
+   ```sh
+   # Set the container image
+   $ helm install --name botkube --namespace botkube \
+	   ... other options ...
+	   --set image.repository=your_account/botkube \
+	   --set image.tag=latest
+	   ...
+	   helm/botkube
+   ```
+
+### Build and run BotKube locally
+1. Build BotKube binary  
+   If you don't want to build the container image, you can build the
+   binary like this,
+   ```sh
+   # Fetch the dependencies
+   $ dep ensure
+   # Build the binary
+   $ go build ./cmd/botkube/
+   ```
+2. Modify `config.yaml` according to your needs
+   ```sh
+   # From project root directory
+   xdg-open config.yaml
```

Comment:
Please add `$`

Reply:
If we add $ in the instructions then it would be hard to copy paste them, places where we just want to show what is output of a particular command we can have the `$`. What do you think?

Reply:
Will make the change consistent once we decide to use one of the way (Right now I have $ at some places and it's missing at some places)

Reply:
I was only concerned about visual experience and consistency across of file. Here, if we just set convention that line starting with `$` is command , user has that in his mindset. Copy-paste I really don't think would make much difference as such. 
I am fine with/without both. 
 https://github.com/infracloudio/botkube/blob/606956d2607de5846e7529b9b22d0bf64bf05a78/CONTRIBUTING.md

Reply:
Yeah, considering the short length of commands, let's add `$` before all the commands. It's easy to type this commands by hand instead of copy paste.